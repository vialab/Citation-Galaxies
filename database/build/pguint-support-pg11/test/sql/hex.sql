SELECT to_hex(95::int4);
SELECT to_hex(-95::int4);
SELECT to_hex(95::int8);
SELECT to_hex(-95::int8);

SELECT to_hex(95::uint4);
SELECT to_hex(95::uint8);
