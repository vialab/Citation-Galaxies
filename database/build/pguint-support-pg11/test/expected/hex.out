SELECT to_hex(95::int4);
 to_hex 
--------
 5f
(1 row)

SELECT to_hex(-95::int4);
  to_hex  
----------
 ffffffa1
(1 row)

SELECT to_hex(95::int8);
 to_hex 
--------
 5f
(1 row)

SELECT to_hex(-95::int8);
      to_hex      
------------------
 ffffffffffffffa1
(1 row)

SELECT to_hex(95::uint4);
 to_hex 
--------
 5f
(1 row)

SELECT to_hex(95::uint8);
 to_hex 
--------
 5f
(1 row)

